src/Raft.sol

```diff
@@ -118,6 +118,10 @@ contract Raft is
     address[] memory admins,
     bool[] memory isActive
   ) public virtual {
+    require(
+      admins.length > 0 && isActive.length > 0,
```

Comment:
They also have to be of equal length, right?

Reply:
Could u adjust error message accordingly. It can be something generic as "invalid inputs or lengths" for example

Reply:
@rahulrumalla @grp06 Why would `addAdmins()` still feature a flag `isActive`? I thought `addAdmins`/`removeAdmins` would remove the need for a separate flag for distinguishing "add"/"remove"?

Reply:
> 

yes - that's like 4 lines below where you commented. But based on Stefan's comment correctly points out, we won't need that